src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferDetail.cshtml.cs

```diff
@@ -14,6 +15,8 @@ public class LocalOfferDetailModel : HeaderPageModel
     private readonly IOrganisationClientService _organisationClientService;
     private readonly IIdamsClient _idamsClient;
     public ServiceDto LocalOffer { get; set; } = default!;
+    public List<AttendingType> ServiceScheduleAttendingTypes { get; set; } = default!;

```

Comment:
Personal preference but prefer empty collections rather than nulls. Makes it easier to work with so you don't have to check for null collections as well as empty - just a thought though. Can use new List<AttendingType>()).

Reply:
Agree having a non-nullable property and setting it to default is not good, but creating an empty collection just to have it unreferenced and garbage collected is not great either.
I've made it nullable and explicitly handle null (in the view).